### DISCUSSION TITLE
Support Chatbot for Flowise itself

### DISCUSSION THREAD

**Discussion by mikehudson2** [FlowiseAI/Flowise#2273] [General]
Has anyone created a chatbot to query the Flowise github project itself? Given the documentation isn't perfect and Flowise is evolving fast that could be really useful - especially for low code guys like me.

**Comment by mad-sol-dev** [comment]
Hi!
I've created one a couple of days ago, it seems to work ok so far. You have to disconnect the "document"-wires after setting up the pinecone vector-database. Otherwise it will always try to fetch the repository and the docs - github will throw a "rate-limit" at some point (most likely this can be improved somehow).

However, the limitations of the built-in chat do make a non-“one-shot” question style somewhat uncomfortable..

Edit: You still have to poke it with “Please base your answer on the documentation and source code.” to get fact-based information.

[GPT4 Flowise Dev Kate Chatflow.json](https://github.com/FlowiseAI/Flowise/files/15144217/GPT4.Flowise.Dev.Kate.Chatflow.json)
[Conversation.txt](https://github.com/FlowiseAI/Flowise/files/15144198/Conversation.txt)
